content/en/docs/reference/command-line-tools-reference/kube-proxy-config.v1alpha1.md

```diff
@@ -0,0 +1,498 @@
+
+
+
+
+
+---
+title: kubeproxy.config.k8s.io/v1alpha1
```

Comment:
i don't object to this, but something to note that kubernetes component config (kube-proxy, kubelet ...etc) APIs change every release. so this file has to be updated every release.

the only component that doesn't do this is kubeadm. kubeadm releases a new API version when something changes.

i assume you don't want to link to godoc:
https://pkg.go.dev/k8s.io/kube-proxy/config/v1alpha1

due to the old argument that the format of godoc is less readable?

Reply:
@neolit123 Yes, these component configs may change every release, just at their command line flags. We'll generate the config along with the command line flags as part of the release process for each version.

godoc is not for YAML writers. If we link to godoc, we have to pin to a release tag because the godoc content is a moving target as well.